src/editorialreview/views.py

```diff
@@ -438,7 +443,18 @@ def editorial_review(request, review_id):
                 default='The publishers',
             )
 
-            for editor in submission.book_editors.all():
+            try:
+                series_editor_email = [submission.series.editor.email]
+            except:
```

Comment:
Can we have a more detailed exception?

Reply:
I agree with @fradeve . Just having except is not that useful, if something goes wrong, we should at least log it